specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -0,0 +1,1049 @@
+  pct2ResTime?: float64;
+
+  @doc("99 percentile response time.")
+  @visibility("read")
+  pct3ResTime?: float64;
+
+  @doc("Throughput.")
+  @visibility("read")
+  throughput?: float64;
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style" //KBytes is expected
+  @doc("Received network bytes.")
+  @visibility("read")
+  receivedKBytesPerSec?: float64;
+
+  #suppress "@azure-tools/typespec-azure-core/casing-style" //KBytes is expected
+  @doc("Send network bytes.")
+  @visibility("read")
+  sentKBytesPerSec?: float64;
+}
+
+@doc("Collection of test run artifacts")
+model TestRunArtifacts {
+  @doc("The input artifacts for the test run.")
+  @visibility("read")
+  inputArtifacts?: TestRunInputArtifacts;
+
+  @doc("The output artifacts for the test run.")
+  outputArtifacts?: TestRunOutputArtifacts;
+}
+
+@doc("The input artifacts for the test run.")
+model TestRunInputArtifacts {
+  @doc("File info")
+  configFileInfo?: TestRunFileInfo;
+
+  @doc("File info")
+  testScriptFileInfo?: TestRunFileInfo;
+
+  @doc("File info")
+  userPropFileInfo?: TestRunFileInfo;
+
+  @doc("File info")
+  inputArtifactsZipFileInfo?: TestRunFileInfo;
+
+  @added(APIVersions.v2023_04_01_preview)
+  @doc("The config json file for url based test")
+  urlTestsConfigFileInfo?: TestRunFileInfo;
+
+  @doc("Additional supported files for the test run")
+  @visibility("read")
+  additionalFileInfo?: TestRunFileInfo[];
+}
+
+@doc("The output artifacts for the test run.")
+model TestRunOutputArtifacts {
+  @doc("File info")
+  resultFileInfo?: TestRunFileInfo;
+
+  @doc("File info")
+  logsFileInfo?: TestRunFileInfo;
+
+  @doc("The container for test run artifacts.")
+  @added(APIVersions.v2023_04_01_preview)
+  artifactsContainerInfo?: ContainerInfo;
+}
+
+@doc("Container info.")
+@added(APIVersions.v2023_04_01_preview)
+model ContainerInfo {
+  @doc("Container URL.")
+  url?: string;
+
+  @doc("Expiry time of the container (ISO 8601 literal format)")
+  expireDateTime?: utcDateTime;
+}
+
+@doc("Collection of test runs")
+@friendlyName("TestRunsList")
+model TestRunsList is Azure.Core.Page<TestRun>;
+
+@doc("Represents collection of metric namespaces.")
+model MetricNamespaceCollection {
+  @doc("The values for the metric namespaces.")
+  value: MetricNamespace[];
+}
+
+@doc("Metric namespace class specifies the metadata for a metric namespace.")
+model MetricNamespace {
+  @doc("The namespace description.")
+  description?: string;
+
+  @doc("The metric namespace name.")
+  name?: string;
+}
+
+@doc("Represents collection of metric definitions.")
+model MetricDefinitionCollection {
+  @doc("the values for the metric definitions.")
+  value: MetricDefinition[];
+}
+
+@doc("Metric definition")
+model MetricDefinition {
+  @doc("List of dimensions")
+  dimensions?: NameAndDesc[];
+
+  @doc("The metric description")
+  description?: string;
+
+  @doc("The metric name")
+  name?: string;
+
+  @doc("The namespace the metric belongs to.")
+  `namespace`?: string;
```

Comment:
Why is this between quotes?

Reply:
because namespace is a keyword in TypeSpec.